bean-validation-custom-constraint/pom.xml

```diff
@@ -237,15 +237,15 @@
             given to the generated war, and hence the context root) -->
         <finalName>${project.artifactId}</finalName>
 
-        <!-- Define plugin versions required for Deployment and Testing Tools -->
+        <!-- Define plug-in versions required for Deployment and Testing Tools -->
         <pluginManagement>
             <plugins>
-                <!-- The Maven Surefire plugin tests your application. Here we ensure we are using a version compatible with Arquillian -->
+                <!-- The Maven Surefire plug-in tests your application. Here we ensure we are using a version compatible with Arquillian -->
                 <plugin>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${version.surefire.plugin}</version>
                 </plugin>
-                <!-- The JBoss AS plugin deploys your war to a local JBoss EAP container -->
+                <!-- The WildFly plug-in deploys your war to a local JBoss EAP container -->
                 <!-- To use, set the JBOSS_HOME environment variable and run:
```

Comment:
Just a doubt: Do we still using JBOSS_HOME even for EAP and WildFly in the docs ?